scripts/ubuntu-bionic/30-timesyncd.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -exv
+
+# If we are an EC2 instance, use the internal amazon NTP service
+if "$WORKER_RUNNER_PROVIDER" == "aws"; then
```

Comment:
This isn't one of the worker-runner scripts, so let's either use a different variable here or change `WORKER_RUNNER_PROVIDER` to be more general to cloud providers.